### ISSUE TITLE
Download for v1.5.10 Windows 64bit is invalid

### ISSUE THREAD

**Issue by HayesSolutions** [yakyak/yakyak#1272]
Hello

In case you are unaware, the yakyak-1.5.10-win32-x64.zip file is invalid and Windows cannot open the folder.

Thanks and kind regards

Dominic

**Comment by HomerSp** [comment]
Duplicate of #1269

**Comment by averissimo** [comment]
@HayesSolutions can you test current v1.5.11 pre-release?

Please don't mind it being called v1.5.10 on the file name, I'm sorting that out before making it an official release

https://github.com/yakyak/yakyak/releases/tag/v1.5.11

**Comment by HayesSolutions** [comment]
> 
> 
> @HayesSolutions can you test current v1.5.11 pre-release?
> 
> Please don't mind it being called v1.5.10 on the file name, I'm sorting that out before making it an official release
> 
> https://github.com/yakyak/yakyak/releases/tag/v1.5.11

Thank you for following this up.  Unfortunately, ths produces the following error
![image](https://user-images.githubusercontent.com/57859813/101927086-fa611b80-3bcb-11eb-91de-51cdd9bcc299.png)
Kind regards
Dominic

**Comment by averissimo** [comment]
Oh man. That's bad, I'll check out on my old windows pc.

Has the problem with the file being invalid been corrected?

**Comment by HayesSolutions** [comment]
> 
> 
> Oh man. That's bad, I'll check out on my old windows pc.
> 
> Has the problem with the file being invalid been corrected?

Hello
No worries, these things happen.  I don't think the file has been corrected yet and I have reverted to 1.5.9 for the time being.  I'd like to update as this version does not always show when I have unread messages and that causes me problems at times!
Best regards
Dominic

**Comment by HomerSp** [comment]
@averissimo The file looks to be generated as a zip for me now, so that should be working at least.

@HayesSolutions Any chance you can post the output from the console (ctrl+alt+i)? Would help us knowing what's wrong. I'm not able to reproduce it myself.

**Comment by HayesSolutions** [comment]
> 
> 
> @averissimo The file looks to be generated as a zip for me now, so that should be working at least.
> 
> @HayesSolutions Any chance you can post the output from the console (ctrl+alt+i)? Would help us knowing what's wrong. I'm not able to reproduce it myself.

Morning HomerSp

In 1.5.9 the console log is present, see below, but after replacing with 1.5.10 the error occurs before the app has loaded and there is no console available.
1.5.9 Console
lectron Deprecation Warning The 'remote' module is deprecated and will be disabled by default in a future version of Electron. To ensure a smooth upgrade and silence this warning, specify {enableRemoteModule: true} in the WebPreferences for this window.
(anonymous) @ remote.js:20
version.coffee:29 YakYak local version is 1.5.9, released version is 1.5.10
DevTools failed to load SourceMap: Could not load content for file:///C:/Program%20Files/yakyak-win32-x64/resources/app/ui/tagg.js.map: 
DevTools failed to load SourceMap: Could not load content for file:///C:/Program%20Files/yakyak-win32-x64/resources/app/ui/notr.js.map: 
DevTools failed to load SourceMap: Could not load content for file:///C:/Program%20Files/yakyak-win32-x64/resources/app/ui/fnuc.js.map: 
DevTools failed to load SourceMap: Could not load content for file:///C:/Program%20Files/yakyak-win32-x64/resources/app/ui/uri.all.js.map: 
DevTools failed to load SourceMap: Could not load content for file:///C:/Program%20Files/yakyak-win32-x64/resources/app/ui/performance-now.js.map: 

Let me know if there is anything else I might do

Kind regards

Dominic

**Comment by HayesSolutions** [comment]
> 
> 
> @averissimo The file looks to be generated as a zip for me now, so that should be working at least.
> 
> @HayesSolutions Any chance you can post the output from the console (ctrl+alt+i)? Would help us knowing what's wrong. I'm not able to reproduce it myself.

Morning All
OK, I have another answer.  In the past, I have simply unzipped the downloaded file to the C:\Program Files directory, overwriting whatever is there already.  That seems to be the root cause of the issue because when I deleted the C:\Program Files\yakyak-win32-x64 folder entirely and then unzipped, the application works without issue.
If this is a case of RTFM, I apologise.
Kind regards
Dominic
